com.unity.perception/Runtime/GroundTruth/Labelers/CameraLabeler.cs

```diff
@@ -17,6 +18,22 @@ public abstract class CameraLabeler
 
         internal bool isInitialized { get; private set; }
 
+        /// <summary>
+        /// Labelers should set this in their setup to define if they support realtime
+        /// visualization of their data. 
+        /// </summary>
+        protected bool supportsVisualization = false;
```

Comment:
That's how we determine if we are going to create the UI elements for the labeler. We could find a different way to do this probably, but I'm not sure if it will gain us anything. Maybe?